### PR TITLE
[176] Unify output of endpoints

### LINKED ISSUE #176: Create participant endpoint should return the same content as participant details
That's blocked by #175.

When we create a participant:
```
curl -X POST \
  http://localhost:5000/participants/ \
  -H 'Content-Type: application/json' \
  -H 'cache-control: no-cache' \
  -d '{
    "name": "Urszula",
    "lastname": "Wawrzyńczyk",
    "phone": "159092699"
}'
```
we're getting participant details in response:
```
201 CREATED
{
    "participant": {
        "lastname": "Wawrzyńczyk",
        "phone": "159092699",
        "name": "Urszula"
    },
    "message": "Participant created successfully."
}
```
For comparison participant details:
```
curl -X GET \
  http://localhost:5000/participants/56/ \
  -H 'cache-control: no-cache'
```
contains:
```
200 OK
{
    "hacknights": [],
    "lastname": "Wawrzyńczyk",
    "id": 56,
    "email": null,
    "github": "",
    "name": "Urszula",
    "phone": "159092699"
}
```
Please, also, provide unit tests.

### PR DESCRIPTION
**Pull request by w1stler**
fixes #176

### CODE REVIEW

**Comment by w1stler** [comment]
IMO this approach would be also applicable to hacknight endpoint.

File: backend/resources/participant.py [hunk 1]
```diff
@@ -37,7 +37,7 @@ def post(self):
         db.session.commit()
 
         return {"message": "Participant created successfully.",
-                "participant": data}, HTTPStatus.CREATED
+                "participant": participant_schema.dump(participant)}, HTTPStatus.CREATED
```
Comment (stanislawK) [inline backend/resources/participant.py]: Length of this line is ok, if we want to switch to black's default 88 characters. If we want to stick to PEP8 it is to long.

Comment (w1stler) [reply backend/resources/participant.py]: I would stick with black. :)

**Comment by OtisRed** [comment]
@w1stler no description on how to test :P

**Comment by w1stler** [comment]
@OtisRed look into linked issue. ;P

**Comment by w1stler** [comment]
Testing:

- make start
- make populate_database
- login in postman with given username: "/auth/login", copy access_token
- use that token in authorization_header for requests below
-- send "get" request to `participants/1/details/`
-- send `get` request to `participants/`

**Comment by magul** [comment]
Please, resolve conflict @w1stler.

**Comment by magul** [comment]
@w1stler please, resolve conflict.

File: backend/tests/test_participants_list.py [hunk 2]
```diff
@@ -46,9 +46,11 @@ def test_create_participant_when_logged_in(app, client, access_token, new_partic
         participant = schema.dump(Participant.query.first())
 
     assert rv.status_code == HTTPStatus.CREATED
-    assert response["participant"]["email"] == new_participant["email"]
+    assert response["email"] == new_participant["email"]
     for value in new_participant.values():
         assert value in participant.values()
+    for value in [*participant.keys()]:
```
Comment (Maciek246) [inline backend/tests/test_participants_list.py]:
```suggestion
    for value in participant.keys():
```

Comment (w1stler) [reply backend/tests/test_participants_list.py]: Hmm, I don't even recall, why I did this in such manner. Nevertheless, changed.

**Pull request merged**